gammapy/cube/fit.py

```diff
@@ -0,0 +1,78 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import numpy as np
+from . import SkyModelMapEvaluator
+from ..stats import cash
+from ..utils.fitting import fit_minuit
+
+
+class CubeFit(object):
```

Comment:
What about the name?
We have `CubeFit` and `SkyModelMapEvaluator`, which is very disparate.
Maybe `SkyModelMapFit` or `MapFit`?